google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -159,14 +158,9 @@ public static void beforeClass() throws IOException {
             .setLifecycleRules(
                 ImmutableList.of(
                     new BucketInfo.LifecycleRule(
-                        new BucketInfo.LifecycleRule.SetStorageClassLifecycleAction(
-                            StorageClass.NEARLINE),
-                        new BucketInfo.LifecycleRule.LifecycleCondition()
-                            .setNumberOfNewerVersions(3)
-                            .setCreatedBefore(new DateTime(System.currentTimeMillis()))
-                            .setAge(1)
-                            .setIsLive(false)
-                            .setMatchesStorageClass(ImmutableList.of(StorageClass.COLDLINE)))))
+                        new BucketInfo.LifecycleRule.DeleteLifecycleAction(),
```

Comment:
Add an import to reduce the long call for `LifecycleRule.SetStorageClassLifecycleAction` and `LifecycleRule.DeleteLifecycleAction`.